pennylane/ops/channel.py

```diff
@@ -267,6 +267,71 @@ def _kraus_matrices(cls, *params):
         return [K0, K1]
 
 
+class ResetError(Channel):
+    r"""ResetError(p_0, p_1, wires)
+    Single-qubit Reset error channel.
+
+    This channel is modelled by the following Kraus matrices:
+
+    .. math::
+        K_0 = \sqrt{1-p_0-p_1} \begin{bmatrix}
+                1 & 0 \\
+                0 & 1
+                \end{bmatrix}
+
+    .. math::
+        K_1 = \sqrt{p_0}\begin{bmatrix}
+                1 & 0  \\
+                0 & 0
+                \end{bmatrix}
+
+    .. math::
+        K_2 = \sqrt{p_0}\begin{bmatrix}
```

Comment:
@nahumsa Why did you change the definition of the channel? 
Before approving, could you please point me to a reference where this channel is defined? I would like to make sure that the name "reset error" coincides with the standard usage. Thanks!

Reply:
Hi @ixfoduap !  I was reading through the [qiskit's docs](https://qiskit.org/documentation/tutorials/simulators/5_noise_transformation.html) and found a direct reference about the `ResetError` Kraus operators: 
![image](https://user-images.githubusercontent.com/49600259/118862685-63a03580-b8b4-11eb-9e16-e3532cb864ac.png)

This was the main reason that I changed the definition of the channel, do you think that this is reasonable?

Reply:
Perfect! Thanks for the clarification!